pet-store-react-hooks/src/Home/Home.tsx

```diff
@@ -61,7 +64,7 @@ export function Home() {
       <div className="all-pets-card-body">
         {loading && <LoadingIndicator />}
         {error && <ErrorMessage message={error} />}
-        {!error && !loading && (
+        {allPets && petKindsMap && !loading && (
```

Comment:
Why `!loading` ?

Reply:
Forgot about it while i was testing for the pet kinds